micro_mg3_0.F90

```diff
@@ -1022,7 +1045,74 @@ subroutine micro_mg_tend ( &
   deltat  = deltatin
   rdeltat = 1._r8 / deltat
 
+  if (trim(micro_mg_precip_frac_method) == 'in_cloud') then
+     precip_frac_method =  MG_PRECIP_FRAC_INCLOUD
+  else if(trim(micro_mg_precip_frac_method) == 'max_overlap') then
+     precip_frac_method = MG_PRECIP_FRAC_OVERLAP
+  endif
+
+  !===============================================
+  ! set mtime here to avoid answer-changing
```

Comment:
We should change this comment since "here" has changed. I'm not even sure what it meant in the first place. Based on the current code, I would make this comment "Set ice nucleation timescale to deltat at start of microphysics loop"

Reply:
Thanks Kate. Just add it.

Reply:
Hi Jian, sorry I wasn't very clear. I wanted to change the "set time here to avoid answer-changing" comment to the one above. Still time to fix it. :)